scripts/cmake/vcpkg_fixup_pkgconfig.cmake

```
@@ -40,18 +40,15 @@
 
 include(vcpkg_escape_regex_control_characters)
 function(vcpkg_fixup_pkgconfig_check_files pkg_cfg_cmd _file _config _system_libs _ignore_flags)
-    # Setup pkg-config paths
-    set(_VCPKG_INSTALLED_PKGCONF "${CURRENT_INSTALLED_DIR}")
-    set(_VCPKG_PACKAGES_PKGCONF "${CURRENT_PACKAGES_DIR}")
-    
     set(PATH_SUFFIX_DEBUG /debug)
-    set(PKGCONFIG_INSTALLED_DIR "${_VCPKG_INSTALLED_PKGCONF}${PATH_SUFFIX_${_config}}/lib/pkgconfig")
-    set(PKGCONFIG_INSTALLED_SHARE_DIR "${_VCPKG_INSTALLED_PKGCONF}/share/pkgconfig")
-    set(PKGCONFIG_PACKAGES_DIR "${_VCPKG_PACKAGES_PKGCONF}${PATH_SUFFIX_${_config}}/lib/pkgconfig")
-    set(PKGCONFIG_PACKAGES_SHARE_DIR "${_VCPKG_PACKAGES_PKGCONF}/share/pkgconfig")
+    set(PATH_SUFFIX_RELEASE)
+    set(PKGCONFIG_INSTALLED_DIR "${CURRENT_INSTALLED_DIR}${PATH_SUFFIX_${_config}}/lib/pkgconfig")
+    set(PKGCONFIG_INSTALLED_SHARE_DIR "${CURRENT_INSTALLED_DIR}/share/pkgconfig")
+    set(PKGCONFIG_PACKAGES_DIR "${CURRENT_PACKAGES_DIR}${PATH_SUFFIX_${_config}}/lib/pkgconfig")
+    set(PKGCONFIG_PACKAGES_SHARE_DIR "${CURRENT_PACKAGES_DIR}/share/pkgconfig")
 
+    set(BACKUP_ENV_PKG_CONFIG_PATH "$ENV{PKG_CONFIG_PATH}")
     if(ENV{PKG_CONFIG_PATH})
```

```suggestion
    if(DEFINED ENV{PKG_CONFIG_PATH})
```

checking for ENV variables requires DEFINED